<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Процессы:       Род. процесс:  Порожденный проц:    t начальное:    t конечное:             Результат:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>738</width>
         <height>391</height>
        </rect>
       </property>
       <widget class="QLabel" name="label_A">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Процесс</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBar">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>10</y>
          <width>118</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_stat_A">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>10</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Состояние</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBar_2">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>40</y>
          <width>118</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_stat_B">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>40</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Состояние</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_B">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Процесс</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBar_3">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>70</y>
          <width>118</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_stat_C">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>70</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Состояние</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_C">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Процесс</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_F">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Процесс</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_stat_F">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>160</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Состояние</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_stat_E">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>130</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Состояние</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBar_4">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>100</y>
          <width>118</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBar_5">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>130</y>
          <width>118</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_stat_D">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>100</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Состояние</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_D">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Процесс</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBar_6">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>160</y>
          <width>118</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_E">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Процесс</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_K">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>250</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Процесс</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_stat_K">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>250</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Состояние</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_stat_H">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>220</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Состояние</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBar_7">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>190</y>
          <width>118</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBar_8">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>220</y>
          <width>118</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_stat_G">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>190</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Состояние</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_G">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>190</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Процесс</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBar_9">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>250</y>
          <width>118</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_H">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>220</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Процесс</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_par_A">
        <property name="geometry">
         <rect>
          <x>74</x>
          <y>11</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Род.</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_par_B">
        <property name="geometry">
         <rect>
          <x>74</x>
          <y>40</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Род.</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_par_C">
        <property name="geometry">
         <rect>
          <x>74</x>
          <y>69</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Род.</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_par_D">
        <property name="geometry">
         <rect>
          <x>74</x>
          <y>98</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Род.</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_par_G">
        <property name="geometry">
         <rect>
          <x>71</x>
          <y>188</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Род.</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_par_F">
        <property name="geometry">
         <rect>
          <x>71</x>
          <y>159</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Род.</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_par_E">
        <property name="geometry">
         <rect>
          <x>71</x>
          <y>130</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Род.</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_par_H">
        <property name="geometry">
         <rect>
          <x>71</x>
          <y>217</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Род.</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_par_K">
        <property name="geometry">
         <rect>
          <x>72</x>
          <y>246</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Род.</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_t_A">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>14</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>t</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_t_B">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>40</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>t</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_t_C">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>70</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>t</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_t_D">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>100</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>t</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_t_E">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>130</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>t</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_t_F">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>160</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>t</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_t_G">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>190</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>t</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_t_H">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>221</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>t</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_t_K">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>250</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>t</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_R1R2R3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>290</y>
          <width>200</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Начальные значения для R1, R2, R3:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_t_B_2">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>39</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>t</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_t_G_2">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>189</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>t</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_t_A_2">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>13</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>t</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_t_C_2">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>69</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>t</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_t_D_2">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>99</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>t</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_t_K_2">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>249</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>t</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_t_E_2">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>129</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>t</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_t_F_2">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>159</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>t</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_t_H_2">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>220</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>t</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_par_A_2">
        <property name="geometry">
         <rect>
          <x>153</x>
          <y>14</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Род.</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_par_B_2">
        <property name="geometry">
         <rect>
          <x>153</x>
          <y>43</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Род.</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_par_C_2">
        <property name="geometry">
         <rect>
          <x>153</x>
          <y>72</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Род.</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_par_K_2">
        <property name="geometry">
         <rect>
          <x>151</x>
          <y>249</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Род.</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_par_F_2">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>162</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Род.</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_par_E_2">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>133</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Род.</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_par_D_2">
        <property name="geometry">
         <rect>
          <x>153</x>
          <y>101</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Род.</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_par_G_2">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>191</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Род.</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_par_H_2">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>220</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Род.</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_C_resualt">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>70</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Процесс</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_G_resualt">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>190</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Процесс</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_E_resualt">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>130</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Процесс</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_K_resualt">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>250</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Процесс</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_F_resualt">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>160</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Процесс</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_H_resualt">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>220</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Процесс</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_A_resualt">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>10</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Процесс</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_D_resualt">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>100</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Процесс</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_B_resualt">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>40</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Процесс</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>310</y>
          <width>60</width>
          <height>20</height>
         </rect>
        </property>
        <property name="statusTip">
         <string/>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string>true</string>
        </property>
        <property name="cursorPosition">
         <number>4</number>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>310</y>
          <width>60</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>false</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>310</y>
          <width>60</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>true</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>758</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
